airflow/docker.go

```diff
@@ -542,6 +625,24 @@ func (d *DockerCompose) getWebServerContainerID() (string, error) {
 	return "", err
 }
 
+func (d *DockerCompose) CheckAiflowVersion() (uint64, error) {
```

Comment:
```suggestion
func (d *DockerCompose) checkAiflowVersion() (uint64, error) {
```

if you can, there is a similar logic in the same file, might want to replace that by this function call

Reply:
okay